src/fsharp/Optimizer.fs

```diff
@@ -295,16 +295,21 @@ let [<Literal>] jitOptDefault = true
 
 let [<Literal>] localOptDefault = true
 
-let [<Literal>] crossModuleOptDefault = true
+let [<Literal>] crossAssemblyOptimizationDefault = true
+
+let [<Literal>] debugPointsForPipeRightDefault = true
 
 type OptimizationSettings = 
-    { abstractBigTargets : bool
+    { 
+      abstractBigTargets : bool
       
       jitOptUser : bool option
       
       localOptUser : bool option
       
-      crossModuleOptUser : bool option 
+      debugPointsForPipeRight: bool option
```

Comment:
We are handling debug points in a particular way for `PipeRight` based on this recd field. I guess it's far out of scope to generalize anything and not worth that amount of effort - so it makes sense.